Chat: Add Suggestions How-To (#8679)
DevExpress/devextreme-documentation: concepts/05 UI Components/Chat/45 Configure Suggestions/05 Fill the Chat Input Field with Predefined Messages.md

> To configure suggestion items that fill the Chat's input field, update the [inputFieldText](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#inputFieldText) property in the **suggestions**.[onItemClick](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/#onItemClick) handler:

Reviewer: ```suggestion
To configure suggestion items for the Chat's input field, update the [inputFieldText](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#inputFieldText) property in the **suggestions**.[onItemClick](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/#onItemClick) handler:
```
